### PR TITLE
When in production, change the seed once a day, not once a minute

### PR DESCRIPTION
**Pull request by chucker**

### CODE REVIEW

**Comment by chucker** [comment]
To test, change `index.html` as follows:

```html
    <script src="_framework/blazor.webassembly.js" autostart="false"></script>
    <script>
            Blazor.start({
                environment: "Production"
            });
    </script>
```

**Comment by ricardoboss** [comment]
Please fix the build failures

File: Prolangle/Program.cs [hunk 1]
```diff
@@ -16,7 +16,20 @@
 {
 	var lp = sp.GetRequiredService<LanguagesProvider>();
 	var logger = sp.GetRequiredService<ILogger<GuessGame>>();
-	return new(lp, logger, () => DateTime.Now.Minute);
+
+	var environment = sp.GetRequiredService<IWebAssemblyHostEnvironment>();
+
+	Console.WriteLine($"Running in {environment.Environment} environment");
```
Comment (ricardoboss) [inline Prolangle/Program.cs:22]: I'd rather not use `Console` directly. Instead, I suggest using an `ILogger`

File: Prolangle/Services/GuessGame.cs [hunk 2]
```diff
@@ -11,7 +14,10 @@ public GuessGame(LanguagesProvider provider, ILogger<GuessGame> logger, Func<int
 		var targetLanguage = languages[random.Next(languages.Count)];
 		TargetLanguageId = targetLanguage.Id;
 
-		logger.LogInformation("Target language is {Language}", targetLanguage.Name);
+		if (environment.Environment == "Development")
```
[the remark this answers is not in the document]
Comment (chucker) [reply Prolangle/Services/GuessGame.cs]: I guess I don't see the huge benefit of https://github.com/dotnet/aspnetcore/blob/c53f7e474ba80124fb9134ebf23d1d626a19b458/src/Hosting/Abstractions/src/HostingEnvironmentExtensions.cs but… sure.

File: Prolangle/Program.cs (hunk 1 above)
Comment (chucker) [reply Prolangle/Program.cs:22]: Huh, I thought logging wasn't configured.

File: Prolangle/Program.cs [hunk 3]
```diff
@@ -16,7 +16,20 @@
...
+
+	var seeder = environment.Environment switch
+	{
+		"Development" => DateTime.Now.Ticks,
+		"Production" => DateTime.Today.Ticks,
+		_ => throw new NotImplementedException()
+	};
+
+	return new(lp, logger, () => (int)(seeder % Math.Pow(2, 31)),
```
[the remark this answers is not in the document]
Comment (chucker) [reply Prolangle/Program.cs:31]: Not sure what you mean. I didn't want to move it to the switch because that line was already a bit wordy.

**Pull request merged**